source/Playnite/Converters/PlayTimeToStringConverter.cs

```diff
@@ -61,6 +63,15 @@ public object Convert(object value, Type targetType, object parameter, System.Gl
             }
 
             var hours = minutes / 60;
+            if (parameter is bool formatToDays && formatToDays && hours >= 24)

```

Comment:
Please add unit test for this.

Reply:
Done in https://github.com/JosefNemec/Playnite/pull/3317/commits/9a9c7a30cf2694a9471f371e26b456f70502d085